src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandlerStorage.js

```diff
@@ -0,0 +1,37 @@
+import TeamsHandler from './TeamsHandler';
+
+export default class TeamsHandlerStorage {
+	static get() {
+		const data = localStorage.getItem('player_list');
```

Comment:
why are you using the localStorage?

Reply:
to store players names and pre defined teams